src/actions/breakpoints/index.js

```diff
@@ -113,11 +113,11 @@ export function disableBreakpoint(location: Location) {
  */
 export function toggleAllBreakpoints(shouldDisableBreakpoints: boolean) {
   return async ({ dispatch, getState, client }: ThunkArgs) => {
-    const breakpoints = getBreakpoints(getState());
+    const breakpoints = getBreakpointsArray(getState());
```

Comment:
this is a nit, but i think I prefer `getBreakpointsList` as it is more descriptive than implementation specific